snmp/datadog_checks/snmp/commands.py

```diff
@@ -33,13 +34,17 @@ def snmp_get(config, oids, lookup_mib):
     def callback(  # type: ignore
         snmpEngine, sendRequestHandle, errorIndication, errorStatus, errorIndex, varBinds, cbCtx
     ):
-        var_binds = vbProcessor.unmakeVarBinds(snmpEngine, varBinds, lookup_mib)
+        if lookup_mib:
+            mibViewController = vbProcessor.getMibViewController(snmpEngine)
+            varBinds = [
+                ObjectType(ObjectIdentity(x[0]), x[1]).resolveWithMib(mibViewController, ignoreErrors=False)
+                for x in varBinds
+            ]
 
         cbCtx['error'] = errorIndication
-        cbCtx['var_binds'] = var_binds
+        cbCtx['var_binds'] = varBinds
```

Comment:
@zoedt Minor comment, is there are reason we need to reassign to original varBinds parameter variable? I think it usually better to avoid replacing content of parameter variable.

Maybe we can do something like this instead?

```
        newVarBinds = varBinds
        if lookup_mib:
            mibViewController = vbProcessor.getMibViewController(snmpEngine)
            newVarBinds = [
                ObjectType(ObjectIdentity(x[0]), x[1]).resolveWithMib(mibViewController, ignoreErrors=False)
                for x in varBinds
            ]

        cbCtx['error'] = errorIndication
        cbCtx['var_binds'] = newVarBinds
```

Reply:
good point! that's better to follow up with your suggestion